packages/custom-elements/src/core.ts

```diff
@@ -19,7 +19,7 @@ const hydrateJSON = async (value: unknown) => {
 		if (isSolenoidFunction(value)) {
 			const fnObj = value;
 			const fnPromise = window.__FNS__[fnObj.module] ?? import(fnObj.module);
-			const closurePromises = Promise.all(fnObj.closure.map(hydrateJSON));
+			const closurePromises = Promise.all(fnObj.closure().map(hydrateJSON));
```

Comment:
This won't be needed AFAIKT, because this would otherwise be pretty messy to serialize.